content/agent/network.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Network Traffic
+kind: documentation
+aliases:
+    - /agent/proxy
+further_reading:
+- link: "/logs/"
+  tag: "Documentation"
+  text: Collect your logs
+- link: "/graphing/infrastructure/process"
+  tag: "Documentation"
+  text: Collect your processes
+- link: "/tracing"
+  tag: "Documentation"
+  text: Collect your traces
+---
+
+## Network Security
+
+1. **Traffic is always initiated by the agent to Datadog. No sessions are ever initiated from Datadog back to the agent**
+2. All traffic is sent over SSL
+3. The destination for all agent data is
+    1. **Agents < 5.2.0** `app.datadoghq.com`
+    1. **Agents >= 5.2.0** `<version>-app.agent.datadoghq.com`
+
+This decision was taken after the POODLE problem, now versioned endpoints start with Agent 5.2.0, i.e. each version of the agent hits a different endpoint based on the version of the *Forwarder*.  
+
+* i.e. Agent 5.2.0 hits `5-2-0-app.agent.datadoghq.com`  
+
+As a consequence whitelist `*.agent.datadoghq.com` in your firewalls.
+
+These domains are **CNAME** records pointing to a set of static IP addresses, these addresses can be found at:  
+
+* [https://ip-ranges.datadoghq.com](https://ip-ranges.datadoghq.com)
+
+The information is structured as JSON following this schema: 
+
+```
+{
+    "version": 1,                       // <-- we increment this every time the information is changed
+    "modified": "YYYY-MM-DD-HH-MM-SS",  // <-- the timestamp of the last modification
+    "agents": {                         // <-- in this section the IPs used for the agent traffic intake
+        "prefixes_ipv4": [              // <-- a list of IPv4 CIDR blocks
+            "a.b.c.d/x",
+            ...
+        ],
+        "prefixes_ipv6": [              // <-- a list of IPv6 CIDR blocks
+            ...
+        ]
+    },
+    "webhooks": {                       // <-- same structure as "agents" but this section is not relevant
+        ...                             //     for agent traffic (webhooks delivered by Datadog to the internet)
+    }
+}
+```
+
+## Open Ports
+
+### Agent v6
+
+  * **`8125/udp`**: dogstatsd
+
+  Unless `dogstatsd_non_local_traffic` is set to true. These ports are available on localhost: 
```

Comment:
nit: it's only one port, "This port is available on localhost:"

Reply:
good catch